app/api/routers/breeze_buddy/alerts/handlers.py

```
@@ -0,0 +1,189 @@
+                )
+                logger.info(
+                    f"Alert lead queued: {lead_id} → {phone} " f"(alert={req.alert_id})"
+                )
+            else:
+                failed.append(
+                    {
+                        "phone": phone,
+                        "reason": "create_lead_call_tracker returned None",
+                    }
+                )
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)